packages/framework/src/Component/Cron/CronModuleRun.php

```diff
@@ -8,7 +8,12 @@
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @ORM\Table(name="cron_module_runs")
+ * @ORM\Table(
+ *      name="cron_module_runs",
+ *      indexes={
+ *           @ORM\Index(columns={"started_at", "finished_at"}),
```

Comment:
why is here a composite index for these two columns? Where do we use them together?

Reply:
I created composite index because I didn't deal with it but you have right, composite index in this case is useless, so I separated them. https://github.com/shopsys/shopsys/pull/3048/commits/8e1976e2455895447153e09dbdef8b13afc324b3